clipmenu

```diff
@@ -35,11 +35,11 @@ list_clips() {
 }
 
 if [[ "$CM_LAUNCHER" == rofi-script ]]; then
-    if [[ -z "$@" ]]; then
+    if [[ $# -eq 0 ]]; then
```

Comment:
as mentioned in last review, use `((` for arithmetic comparison in bash, not `[[` :-)

Reply:
Sure, but i'd like to understand why to improve my own scripts in the futue

Reply:
Mostly stylistic reasons. It allows for easy checking of existence/non-existence and we already use this all over the place, for example in clipmenud you can see a lot of `(( some_flag ))`, which is more terse, and easier to skim through.

In general I try to only ever use `[[` for non-arithmetic comparison, and always use `((` for arithmetic since it usually expresses whatever is desired in a more compact and readable way. In this case it's somewhat negligible, but I'd prefer to keep the style we already use :-)

Reply:
OK, makes a lot of sense.